eng/build.sh

```diff
@@ -17,47 +17,74 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
-  echo "  --rebuild                  Rebuild all source projects"
-  echo "  --test                     Build and run tests (short: -t)"
-  echo "  --pack                     Package build outputs into NuGet packages"
-  echo "  --sign                     Sign build outputs"
-  echo "  --publish                  Publish artifacts (e.g. symbols)"
-  echo "  --clean                    Clean the solution"
+  echo "  --build (-b)               Build all source projects."
+  echo "  --clean                    Clean the solution."
+  echo "  --pack                     Package build outputs into NuGet packages."
+  echo "  --publish                  Publish artifacts (e.g. symbols)."
+  echo "  --rebuild                  Rebuild all source projects."
+  echo "  --restore (-r)             Restore dependencies."
+  echo "  --sign                     Sign build outputs."
+  echo "  --test (-t)                Build and run tests."
   echo ""
 
   echo "Libraries settings:"
-  echo "  --framework                Build framework: net5.0 or net472 (short: -f)"
-  echo "  --coverage                 Collect code coverage when testing"
-  echo "  --testscope                Test scope, allowed values: innerloop, outerloop, all"
-  echo "  --testnobuild              Skip building tests when invoking -test"
-  echo "  --allconfigurations        Build packages for all build configurations"
+  echo "  --allconfigurations        Build packages for all build configurations."
+  echo "  --coverage                 Collect code coverage when testing."
+  echo "  --framework (-f)           Build framework: net5.0 or net472."
+  echo "  --testnobuild              Skip building tests when invoking -test."
+  echo "  --testscope                Test scope, allowed values: innerloop, outerloop, all."
   echo ""
 
   echo "Native build settings:"
-  echo "  --clang                    Optional argument to build using clang in PATH (default)"
-  echo "  --clangx.y                 Optional argument to build using clang version x.y"
+  echo "  --clang                    Optional argument to build using clang in PATH (default)."
+  echo "  --clangx.y                 Optional argument to build using clang version x.y."
   echo "  --cmakeargs                User-settable additional arguments passed to CMake."
-  echo "  --gcc                      Optional argument to build using gcc in PATH (default)"
-  echo "  --gccx.y                   Optional argument to build using gcc version x.y"
+  echo "  --gcc                      Optional argument to build using gcc in PATH (default)."
+  echo "  --gccx.y                   Optional argument to build using gcc version x.y."
+  echo ""
 
   echo "Command line arguments starting with '/p:' are passed through to MSBuild."
   echo "Arguments can also be passed in with a single hyphen."
+  echo ""
+
+  echo "Here are some quick examples:"
+  echo ""
+  echo "Build CoreCLR on Linux for x64 on release configuration:"
```

Comment:
```suggestion
  echo "Build CoreCLR for Linux for x64 on release configuration:"
```
It happens you do have to build ON Linux, but you'll discover that quick if you try on Windows.